### PR TITLE
Welcome to conda dot org blog post

### PR DESCRIPTION
**Pull request by travishathaway**
At yesterday's community meeting, we decided it would be a good idea to write a "welcome" blog post for the website. This pull requests adds that blog post.

### CODE REVIEW

File: blog/2023-04-27-welcome-to-conda-dot-org.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+[conda-dot-org-matrix]: https://matrix.to/#/#conda:matrix.org
+[mamba-github]: https://github.com/mamba-org
+[conda-forge-github]: https://github.com/conda-forge
+
+Welcome to [conda.org](https://conda.org)! This website was created by and for the [conda community](/community)
+and aims to support the greater conda ecosystem. As a site visitor, here are just some of the things
+that you can expect to find here going forward:
+
+- **News and updates** for many projects within the conda ecosystem; from organizations such as [conda][conda-github],
+  [mamba][mamba-github] and [conda-forge][conda-forge-github]
```
Comment (kathatherine) [inline blog/2023-04-27-welcome-to-conda-dot-org.mdx:36]:
```suggestion
  [mamba][mamba-github], and [conda-forge][conda-forge-github]
```

File: blog/2023-04-27-welcome-to-conda-dot-org.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- Information on **how to get involved** with our projects (check out the [Community](/community) page)
+- Plus more!
+
+## Going forward
+
+Below are just a couple of the feature requests that we plan on implementing in the coming months:
+
+- An embedded community calendar so we can advertise all conda community events in a single place
+- Blog posts on various aspects of managing [conda][conda-project-github] as well as tutorial articles
```
Comment (kathatherine) [inline blog/2023-04-27-welcome-to-conda-dot-org.mdx:45]:
```suggestion
- Blog posts on various aspects of managing [conda][conda-project-github], as well as tutorial articles
```

File: blog/2023-04-27-welcome-to-conda-dot-org.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Check out even more of our open and planned issues [here][conda-dot-org-github].
+
+## We need you ☝️
+
+Are you involved in a project within the conda ecosystem and want to share your experiences? Does writing a blog
+post about it sound fun? If so, we encourage you to reach out to us via our [GitHub project][conda-dot-org-github]
+(go ahead and create an issue) or contact us directly via our [Matrix chat][conda-dot-org-matrix]. We are currently looking to
```
Comment (kathatherine) [inline blog/2023-04-27-welcome-to-conda-dot-org.mdx:53]:
```suggestion
post about it sound fun? If so, we encourage you to reach out to us via our [GitHub project][conda-dot-org-github],
(go ahead and create an issue), or contact us directly via our [Matrix chat][conda-dot-org-matrix]. We are currently looking to
```

**Review by kathatherine — DISMISSED** [review]
Couple commas. :D

File: blog/2023-04-27-welcome-to-conda-dot-org.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Welcome to conda.org!"
+slug: "2022-04-27-welcome-to-conda-dot-org"
+tags: [announcement]
+description: |
+  Welcome to our new website! In this article, we talk about why we built
+  this website and how we would like to see be used going forward.
```
Comment (beeankha) [inline blog/2023-04-27-welcome-to-conda-dot-org.mdx:7]:
```suggestion
  this website and how we would like to see it be used going forward.
```

We could alternatively put something different here, like:
"... and where we see it going as more gets added to it."

File: blog/2023-04-27-welcome-to-conda-dot-org.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## We need you ☝️
+
+Are you involved in a project within the conda ecosystem and want to share your experiences? Does writing a blog
+post about it sound fun? If so, we encourage you to reach out to us via our [GitHub project][conda-dot-org-github]
+(go ahead and create an issue) or contact us directly via our [Matrix chat][conda-dot-org-matrix]. We are currently looking to
+fill our blog with a diverse set of entries from the entire conda community.
+
+If you have ideas about new features we could add to this website or want to become a volunteer, please reach out
+to us too.
```
Comment (beeankha) [inline blog/2023-04-27-welcome-to-conda-dot-org.mdx:57]:
```suggestion
to us.
```

File: blog/2023-04-27-welcome-to-conda-dot-org.mdx (hunk 4 above)
Comment (travishathaway) [reply blog/2023-04-27-welcome-to-conda-dot-org.mdx:7]: oof, yeah I'll re-word this.

**Review by beeankha — APPROVED** [review]
🎉

**Pull request merged**